scylla/src/transport/execution_profile.rs

```diff
@@ -485,4 +485,8 @@ impl ExecutionProfileHandle {
     pub fn map_to_another_profile(&mut self, profile: ExecutionProfile) {
         self.0 .0.store(profile.0)
     }
+
+    pub fn load_balancing_policy(&self) -> Arc<dyn LoadBalancingPolicy> {
+        self.0 .0.load().load_balancing_policy.clone()
+    }
```

Comment:
I'd prefer to have this `pub(crate)`, if possible.

Reply:
A docstring providing rationale behind exposing this would be welcome, then. A consise rephrasing of the above  explanation should do.

Reply:
a484a4c